### PR TITLE
feat: add support to upload and download images

### PR DESCRIPTION
**Pull request by ManemDhanush**
Before
- can only upload and download text

After:
- can now upload any file
- can download any file to downloads folder

Thoughts: 
- should we have both text and media support?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
    npm install express 
    npm install discord.js
    npm install dotenv
+   npm install request
```
Comment (QuickHawk) [inline README.md:16]: This package has been deprecated it seems......

Replace with another package like `axios`

File: src/api/discord/download.js [hunk 2]
```diff
@@ -41,7 +43,12 @@ async function download(title) {
             // reversing as the latest message will be featched first
             const reversedMessages = messages.reverse();
             reversedMessages.forEach(msg => {
-                console.log(`${msg.author.tag}: ${msg.content}`);
+                // console.log(`${msg.author.tag}: ${msg.content}`);
+                // console.log(msg.attachments);
+                if(msg.attachments.size > 0) {
+                    const fileUrl = msg.attachments.first().url;
+                    request(fileUrl).pipe(fs.createWriteStream(`./downloads/${title}.jpg`));
+                }
```
Comment (QuickHawk) [inline src/api/discord/download.js:57]: Add a console message for user to understand that the download has been completed and can be located <location_name>.

File: src/api/discord/upload.js [hunk 3]
```diff
@@ -42,7 +42,7 @@ async function upload(filePath, title) {
         await thread.join();
         // Send a confirmation message
         await thread.send(`Created and joined thread: ${thread.name}`);
-        await thread.send(filePath); // yet add attachment support
+        await thread.send({ files: [filePath] }); // yet add attachment support
```
Comment (QuickHawk) [inline src/api/discord/upload.js]: From where the shit are you getting `filePath` man? I don't see any variable declared.

**Review by QuickHawk — CHANGES_REQUESTED** [review]

File: src/api/discord/upload.js (hunk 3 above)
Comment (ManemDhanush) [reply src/api/discord/upload.js]: Check the method declaration, it'll be passed as a param

**Review by QuickHawk — APPROVED** [review]

**Pull request merged**